submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListValidationTest.kt

```diff
@@ -90,19 +181,34 @@ class FileListValidationTest(
 
         val exception = assertThrows(WebClientException::class.java) { webClient.validateFileList(fileList.name) }
         assertThat(exception.statusCode).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR)
-        val expectedError = jsonObj {
+        val expectedNfsError = jsonObj {
+            "log" to jsonObj {
+                "level" to "ERROR"
+                "message" to """
+                        The following files could not be found:
+                          - Plate1.tif
+                        List of available sources:
+                          - biostudies-mgmt-filelist-v@ebi.ac.uk user files
+                """.trimIndent()
+                "subnodes" to jsonArray()
+            }
+            "status" to "FAIL"
+        }
+        val expectedFireError = jsonObj {
             "log" to jsonObj {
                 "level" to "ERROR"
                 "message" to """
                         The following files could not be found:
                           - Plate1.tif
                         List of available sources:
                           - biostudies-mgmt-filelist-v@ebi.ac.uk user files
+                          - EBI internal files Archive
                 """.trimIndent()
                 "subnodes" to jsonArray()
             }
             "status" to "FAIL"
         }
+        val expectedError = if (enableFire) expectedFireError else expectedNfsError
```

Comment:
I would rather create two test, `file list with missing files when NFs` and `file list with missing files When fire`, conditional checks in test code just make it hard to follow it. (So in this case I think is better to favor simplicity over code re use)